### PR TITLE
Added priv_distributeRawTransaction

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added priv_distributeRawTransaction

### CODE REVIEW

File: docs/HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md [hunk 1]
```diff
@@ -7,20 +7,90 @@ Create and send private transactions using:
...
+    
+    Send privacy marker transaction using `eth_sendTransaction` with EthSinger: 
+    
+    ```json
+    {
+      "jsonrpc":"2.0",
+      "method":"eth_sendTransaction",
+      "params":[{
+        "from": "0xfe3b557e8fb62b89f4916b721be55ceb828dbd73",
+        "data": "0xfd0d90ab824574abc19c0776ca0210e764561d0ef6d621f2bbbea316eccfe56b",
```
Comment (NicolasMassart) [inline docs/HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md]: may we explain the data example?

File: docs/HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md [hunk 2]
```diff
@@ -7,20 +7,90 @@ Create and send private transactions using:
...
+## eea_sendRawTransaction 
+
+[`eea_sendRawTransaction`](../../Reference/API-Methods.md#eea_sendrawtransaction) distributes the private transaction to the participating 
+nodes and, signs and submits the [privacy marker transaction] as described in [Private Transaction Processing](../../Concepts/Privacy/Private-Transaction-Processing.md).   
+
+## priv_distributeRawTransaction 
+
+[`priv_distributeRawTransaction`](../../Reference/API-Methods.md#priv_distributerawtransaction) distributes the 
+private transaction to the participating noes but does not sign and submit the [privacy marker transaction](../../Concepts/Privacy/Private-Transaction-Processing.md).
+That is, [`priv_distributeTransaction`](../../Reference/API-Methods.md#priv_distributerawtransaction)
```
[the remark this answers is not in the document]
Comment (MadelineMurray) [reply docs/HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md]: Fixing in another PR

File: docs/HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md [hunk 3]
```diff
@@ -7,20 +7,90 @@ Create and send private transactions using:
...
+
+When using [`priv_distributeRawTransaction`](../../Reference/API-Methods.md#priv_distributerawtransaction) 
+instead of [`eea_sendRawTransaction`](../../Reference/API-Methods.md#eea_sendrawtransaction), send the 
+enclave key returned by [`priv_distributeRawTransaction`](../../Reference/API-Methods.md#priv_distributerawtransaction) 
+as the `data` in a [public Ethereum transaction](Transactions.md). That is, you sign and submit the privacy marker transaction yourself. 
+
+Signing and submitting the [privacy marker transaction] instead of having it signed by the Besu node 
+when processing the private transaction enables greater control over the signing of the [privacy marker transaction]. 
+
+!!! note
```
[the remark this answers is not in the document]
Comment (MadelineMurray) [reply docs/HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md]: Fixing in another PR

File: docs/HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md]: Will address this after 1.3

**Pull request merged**